gcc/rust/typecheck/rust-hir-type-check-expr.cc

```diff
@@ -272,7 +272,8 @@ TypeCheckExpr::visit (HIR::CompoundAssignmentExpr &expr)
   if (!valid)
     {
       rust_error_at (expr.get_locus (),
-		     "cannot apply this operator to types %s and %s",
+		     "cannot apply operator %<%s%> to types %s and %s",
+		     expr.get_operator_str ().c_str (),
```

Comment:
```suggestion
		     "cannot apply operator %qs to types %s and %s",
		     expr.get_operator_str ().c_str (),
```

`%qs` is a shorthand for the same thing :)